### PR TITLE
amfora.1

### PR DESCRIPTION
**Pull request by concussious**
Here is some in-band documentation in mdoc. I think you should accept my pull request because Amfora is in your terminal. Man pages are the standard way of documentation in the terminal for generations. I think this is the lightest, fastest, most predictable and most reliable form of documentation. I am willing to maintain this man page.

### CODE REVIEW

**Comment by makeworld-the-better-one** [comment]
I appreciate the effort and understand that having no manpage can be confusing. But I worry about having to maintain a manpage and a wiki in sync, this isn't something I'd like to do. Would it be ok to instead just have a link to the wiki when you run `amfora --help`?

**Comment by concussious** [comment]
That still doesn't cover the case that someone is look for a man page before they decide if they're going to start guessing.

It's your software, so of course you can do whatever you want. It's a nice browser, thank you for writing it. Man pages are much more flexible than the wiki, just as easy to add to, more standard (the standard), much, more lightweight, and available offline. One of the biggest strengths of *nix is that documentation is handled in a reliable, predictable way. 

I believe my second commit is close to a minimum viable manpage if you want to go the route of minimum viable manpage.
I also believe my latest commit is (still unfinished but very close) more comprehensive than the wiki.

Why would you rather have a wiki than a man page? They both require a github account to add to. Man page is universal. It's <8kb. It's always in sync with the software. Microsoft doesn't track you every time you open it. If your connection is metered it's much more useful. It's a huge privacy win.

Please consider this is the only gemini browser in the Arch Linux repo. I've heard people in a few different contexts say they don't bother with software if there's no manpage.

File: amfora.1 [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+.Sh HISTORY
+.Nm
+v1.0 was released on GitHub June 1, 2020 with tabs, TOFU, colors, history and input.
+Bookmarks, charsets, and search were added in v1.1 on Jun 24.
+Downloading was implemented in v1.3 on Jul 2. Themeing came in v1.4 on Jul 28.
+Emojies and extended proxy support came in v1.5 on Sep 01.
+Client certificates were implemented in v1.6 on Nov 4.
+Subscriptions were implemented in v1.7 on Dec 20, and a GitHub wiki was also started.
+v1.8 added media type handlers on Feb 17, 2021.
+v1.9 began storing bookmarks in XBEL on Dec 7.
```
Comment (aabacchus) [inline amfora.1:244]: If you think this is necessary information, you might put in in a `.Bd -bullet` or something to separate each sentence onto separate lines.

File: amfora.1 [hunk 2]
```diff
@@ -0,0 +1,264 @@
+.Dd $Mdocdate$
+.Dt AMFORA 1
+.Os
+.Sh NAME
+.Nm amfora
+.Nd fancy terminal gemini browser
+.Sh SYNOPSIS
+.Nm amfora
+.Op Ar gemini-url
```
[the remark this answers is not in the document]
Comment (concussious) [reply amfora.1]: Fixed, thank you.

File: amfora.1 [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+Visit subscription feed
+.It Cm about:manage-subscriptions
+Visit unsubscribe list
+.It Va foo
+Search the geminispace for
+.Va foo .
+.El
+.Ss BOOKMARKS
+.Nm
+stores it's bookmarks in an XML format called XBEL, located at
```
[the remark this answers is not in the document]
Comment (concussious) [reply amfora.1]: Fixed, thank you.

File: amfora.1 (hunk 1 above)
Comment (concussious) [reply amfora.1]: I think history is never necessary, but it always adds quality. Especially in the form of man pages, where this information is so condensed, it's really excellent. In my opinion, the quality/bloat factor of adding history to a man page is very good. Also, history is a standard section in the man page hierarchy.

 For some reason, I've never been able to make .Bl -bullet work, so I used .Bl -tag instead. Thanks again.

**Comment by makeworld-the-better-one** [comment]
I'm sorry but I'm not interested in maintaining both a man page and wiki at the moment. Ideally Amfora would have a man page, but I can't commit to it right now.

**Comment by aabacchus** [comment]
Given the rate of development (3 commits in nearly 7 months), I am willing to maintain a manpage alongside the wiki if that's your worry. Alternatively, you could always add a note to the manpage saying, "This reference may be out-of-date; see the wiki at https://.../ for up-to-date documentation".

**Comment by makew0rld** [comment]
@aabacchus @concussious if either of you open to maintaining the man page and add that note I'd be happy to merge this, thanks.

If this PR is

**Comment by concussious** [comment]
Thanks @makew0rld!

**Pull request merged**